src/redisjson.rs

```diff
@@ -11,6 +11,7 @@ use crate::formatter::RedisJsonFormatter;
 use crate::nodevisitor::{StaticPathElement, StaticPathParser, VisitStatus};
 use crate::REDIS_JSON_TYPE_VERSION;
 
+use crate::error::Error::{GeneralError, PathNotAnObject, WrongStaticPath};
```

Comment:
I think it would be more readable just to `use crate::error::Error` here and then explicitly write `Err(Error::GeneralError)` etc. below.